### PR TITLE
Get Breez service status

### LINKED ISSUE #433: Add a global status API
In case of system is down, general health check.

### PR DESCRIPTION
**Pull request by dangeross**
This PR adds an interface to fetch the Breez service status.

Builds on #627 
Fixes #433

### CODE REVIEW

**Review by erdemyerebasmaz — COMMENTED** [review]
Please expose `fetchBreezStatus` through Flutter plugin on `libs/sdk-flutter/lib/breez_sdk.dart`
```
/// Fetch the current Breez service status.
Future<BreezStatusResponse> fetchBreezStatus() async => await _lnToolkit.fetchBreezStatus();
```
Could make a new segment called 
`/* Support API's */`

**Comment by dangeross** [comment]
> Please expose `fetchBreezStatus` through Flutter plugin on `libs/sdk-flutter/lib/breez_sdk.dart`

Thanks for the tips @erdemyerebasmaz. Just to be clear, should I do this every time a new interface is added?

**Comment by erdemyerebasmaz** [comment]
> > Please expose `fetchBreezStatus` through Flutter plugin on `libs/sdk-flutter/lib/breez_sdk.dart`
> 
> Thanks for the tips @erdemyerebasmaz. Just to be clear, should I do this every time a new interface is added?

@dangeross Yes, it's not always required but as a rule of thumb any changes made or added to `bindings.rs` needs to be reflected here.

File: tools/sdk-cli/src/command_handlers.rs [hunk 1]
```diff
@@ -433,6 +433,10 @@ pub(crate) async fn handle_command(
                 _ => Err(anyhow!("Unexpected result type")),
             }
         }
+        Commands::FetchBreezStatus {} => {
+            let breez_status = sdk().unwrap().fetch_breez_status().await?;
```
Comment (ok300) [inline tools/sdk-cli/src/command_handlers.rs:437]:
```suggestion
            let breez_status = sdk()?.fetch_breez_status().await?;
```

File: libs/sdk-bindings/src/breez_sdk.udl [hunk 2]
```diff
@@ -814,6 +824,9 @@ interface BlockingBreezServices {
    [Throws=SendOnchainError]
    SendOnchainResponse send_onchain(SendOnchainRequest req);
 
+   [Throws=SdkError]
+   BreezStatusResponse fetch_breez_status();
```
[the remark this answers is not in the document]
Comment (dangeross) [reply libs/sdk-bindings/src/breez_sdk.udl]: Makes sense. Renamed and tested:
```
sdk> service_health_check
{
  "status": "Operational"
}
```

File: tools/sdk-cli/src/command_handlers.rs (hunk 1 above)
Comment (dangeross) [reply tools/sdk-cli/src/command_handlers.rs]: Done

**Review by ok300 — APPROVED** [review]

**Pull request merged**